### PR TITLE
Bumping dependencies version.

### PR DESCRIPTION
**Pull request by vidorteg**
Updating indirect dependencies.

### CODE REVIEW

**Comment by jalissia** [comment]
Shouldn't there be a package.json file change not just the lock file?

**Comment by vidorteg** [comment]
AFAIK package.json just tracks direct dependencies, while package.json.lock tracks indirect dependencies. In this we just needed to update indirect dependencies.

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-edge-devtools",
-    "version": "1.0.4",
+    "version": "1.0.6",
```
Comment (jalissia) [inline package-lock.json]: Shouldn't this be in the package.json file?

Comment (jalissia) [inline package-lock.json]: Shouldn't this be in the package.json file?

Comment (vidorteg) [reply package-lock.json]: it is 😃 , this PR bumped the package.json version:
https://github.com/microsoft/vscode-edge-devtools/commit/f7ab8a7a9d5b14a5732c631f4ab8f290ea154e00

Comment (jalissia) [reply package-lock.json]: Oh i see. So it was actually my fault oops!

**Pull request merged**